courses/fundamentals_of_ada/labs/prompts/230_interfacing_with_c++/io.adb

```diff
@@ -0,0 +1,45 @@
+with Interfaces.C;
+with Interfaces.C.Strings;
+with Ada.Text_IO;
+package body Io is
+   -- Create import for C's "gets" function
+   -- NOTE: "gets" returns the string as both the parameter AND the
+   -- return value, so you need to specify at LEAST the parameter
+   function Read_From_Stdin return String is
+   begin
+      -- Call C's "gets" function and convert the result to an Ada string
```

Comment:
We should replace gets by fgets, gets is not even available in C11 because the C comittee deemed it too dangerous (yes, the C comittee, people that eat segfaults at lunch).